src/modules/admin/admin.route.tsx

```
@@ -21,6 +22,11 @@ export const adminRoutes = [
     path={adminPaths.product}
     element={<Admin />}
   />,
+  <Route
+    id={adminPaths.addproduct}
+    path={adminPaths.addproduct}
+    element={<Admin />}
+  />,
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)